everware/tests/test_repourl_parser.py

```diff
@@ -0,0 +1,40 @@
+from .. import git_executor
+
+def test_parser():
+    tests = [
+        (
+            'git://github.com/aladagemre/django-notification.git@2927346f4c513a217ac8ad076e494dd1adbf70e1',
```

Comment:
`git` the command line program uses `ssh` to establish the identity of the remote machine when you give it a URL like `git@github.com:everware/everware.git`. As all the containers are "fresh" it is as if you connect to a remote machine for the first time via `ssh`: `ssh` asks you to confirm the fingerprint of the machine and confirming the fingerprint isn't possible via a script. There is some more discussion in #77.